GliaWidgets/Coordinator/RootCoordinator.swift

```diff
@@ -219,72 +226,17 @@ class RootCoordinator: SubFlowCoordinator, FlowCoordinator {
 }
 
 extension RootCoordinator {
-    private func presentWindow(bubbleView: BubbleView, animated: Bool) {
-        guard window == nil else { return }
-
-        let window = makeWindow(bubbleView: bubbleView)
-        window.rootViewController = navigationController
-        window.isHidden = false
-        window.transform = CGAffineTransform(scaleX: 0.1, y: 0.1)
-        self.window = window
-        isWindowVisible.value = true
-
-        UIView.animate(
-            withDuration: animated ? 0.4 : 0.0,
-            delay: 0.0,
-            usingSpringWithDamping: 0.8,
-            initialSpringVelocity: 0.5,
-            options: .curveEaseInOut,
-            animations: {
-                window.transform = .identity
-            }
-        )
-    }
-
-    private func dismissWindow(animated: Bool, completion: @escaping () -> Void) {
-        guard let window = window else { return }
-        isWindowVisible.value = false
-        UIView.animate(
-            withDuration: animated ? 0.5 : 0.0,
-            delay: 0.0,
-            options: .curveEaseInOut,
-            animations: { window.alpha = 0.0 },
-            completion: { _ in completion() }
-        )
-    }
-
-    private func makeWindow(bubbleView: BubbleView) -> GliaWindow {
-        if #available(iOS 13.0, *) {
-            if let windowScene = windowScene() {
-                return GliaWindow(
-                    bubbleView: bubbleView,
-                    windowScene: windowScene,
-                    delegate: self
-                )
-            } else {
-                return GliaWindow(
-                    bubbleView: bubbleView,
-                    delegate: self
-                )
-            }
-        } else {
-            return GliaWindow(
-                bubbleView: bubbleView,
-                delegate: self
-            )
+    private func presentWindow(animated: Bool, completion: (() -> Void)? = nil) {
```

Comment:
we're not really presenting anything on window here, maybe change the function name?

Reply:
Good catch. Leftover name